bundler/spec/dependabot/bundler/file_fetcher/require_relative_finder_spec.rb

```diff
@@ -65,7 +65,7 @@
       end
```

Comment:
I don't love how this reads, I would suggest removing the nesting here:

```ruby
    context "with a file that does include a relative path" do
      # ...
      context "when it needs to be evaled" do
        let(:file_body) do
          'require_relative "./my_file_#{raise %(hell)}"'
        end
        it { is_expected.to eq([]) }
      end

      context "when it can't be evaled" do
        let(:file_body) do
          'require_relative "./my_file_#{unknown_var}"'
        end
        it { is_expected.to eq([]) }
      end
```